src/components/exercises/App1/ProfileSettings.js

```diff
@@ -0,0 +1,69 @@
+import React, { useState } from "react";
+import { useDispatch, useSelector } from "react-redux";
+import { profileThunk } from "./redux/slices/userSlice";
+
+const ProfileSettings = () => {
+  const dispatch = useDispatch();
+  const messageSubmit = useSelector((selector) => selector.user.submitedData);
+  const [clicked, setClicked] = useState(false);
+  const [firstName, setFirstName] = useState("");
```

Comment:
Profile data should be filled with logged in user data which comes from the store if the user is logged in.